### PR TITLE
feat: sentry

### PR DESCRIPTION
**Pull request by TorstenDittmann**
## What does this PR do?

- adds sentry to prod and stage
- uploads sourcemaps

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

✅

### CODE REVIEW

File: src/hooks.client.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { handleErrorWithSentry, replayIntegration } from "@sentry/sveltekit";
+import * as Sentry from '@sentry/sveltekit';
+
+Sentry.init({
+  dsn: 'https://27d41dc8bb67b596f137924ab8599e59@o1063647.ingest.us.sentry.io/4507497727000576',
```
Comment (Meldiron) [inline src/hooks.client.ts:5]: Due to this, both Stage and Production will go to same project. We might want another ENV variable for DSN

File: src/hooks.server.ts [hunk 2]
```diff
@@ -1,8 +1,14 @@
+import * as Sentry from '@sentry/sveltekit';
 import type { Handle } from '@sveltejs/kit';
 import redirects from './redirects.json';
 import { sequence } from '@sveltejs/kit/hooks';
 import { BANNER_KEY } from '$lib/constants';
 
+Sentry.init({
+    dsn: "https://27d41dc8bb67b596f137924ab8599e59@o1063647.ingest.us.sentry.io/4507497727000576",
```
Comment (Meldiron) [inline src/hooks.server.ts:8]: Same here

File: src/hooks.client.ts (hunk 1 above)
Comment (TorstenDittmann) [reply src/hooks.client.ts:5]: should be fine, we can make it dependend on the url 👍🏻

File: src/hooks.server.ts (hunk 2 above)
Comment (Meldiron) [reply src/hooks.server.ts:8]: If we dont do it as configurable variable from ENV (GitHub secrets), we should make it const and re-use, so we can update it in 1 place in future

File: src/routes/sentry-example/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<!-- 
+This is just a very simple page with a button to throw an example error.
+Feel free to delete this file and the entire sentry route.
```
Comment (Meldiron) [inline src/routes/sentry-example/+page.svelte:3]: If we plan to keep it, we can remove this line of the comment

File: src/routes/sentry-example/+server.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+// This is just a very simple API route that throws an example error.
+// Feel free to delete this file and the entire sentry route.
```
Comment (Meldiron) [inline src/routes/sentry-example/+server.js:2]: Same here, no need for this line

File: src/routes/sentry-example/+page.svelte (hunk 3 above)
Comment (TorstenDittmann) [reply src/routes/sentry-example/+page.svelte:3]: just wanna keep it for testing stage 👍🏻

**Review by Meldiron — APPROVED** [review]

**Pull request merged**